src/components/Banner/Banner.tsx

```diff
@@ -1,21 +1,31 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, HTMLProps } from "react";
 import classnames from "classnames";
 
 import "./_banner.scss";
 
-export default function Banner({ isActive = false, children, type }) {
+type Props = {
+  isActive: boolean;
+  children: any;
+  variant: "positive" | "caution" | "negative";
+} & HTMLProps<HTMLElement>;
+
+export default function Banner({
+  isActive,
+  children,
+  variant,
```

Comment:
`type` was changed to `variant` here and in the tests but not in `layout.js`

Reply:
D'oh